augurpy/read_load.py

```diff
@@ -16,21 +22,53 @@ def load(input: Union[AnnData, DataFrame], meta: DataFrame, label_col: str, cell
         cell_type_col: column of the meta DataFrame or the Anndata or matrix containing the cell type labels for each
             cell in the cell-by-gene expression matrix
 
-
     Returns:
-        Pandas DataFrame containing gene expression values (cells in rows, genes in columns), cell type and condition
+        Anndata object containing gene expression values (cells in rows, genes in columns) and cell type, label as obs
     """
-    pass
+    if isinstance(input, AnnData):
+        input.obs = input.obs.rename(columns={cell_type_col: "cell_type", label_col: "label"})
+        out = input
+
+    elif isinstance(input, DataFrame):
+        try:
+            cell_type = input[cell_type_col]
+            label = input[label_col]
+        except KeyError:
+            print("No column names matching cell_type_col and label_col. Looking in meta data.")
```

Comment:
I'd suggest to use https://rich.readthedocs.io/en/stable/introduction.html here 

```suggestion
            print("[bold yellow]No column names matching cell_type_col and label_col. Looking in meta data.")
```

I usually do errors in red, warnings in yellow and successes in green.